apps/WyndDEX/app.json

```
@@ -0,0 +1,11 @@
+{
+  "appCategory": "DeFi",
+  "appName": "WyndDEX",
+  "appSummary": "Cross Chain DeFi Hub.",
+  "appWebsiteUrl": "https://app.wynddao.com/",
+  "externalUrls": {
+    "twitter": "https://twitter.com/wynddao",
+    "github": "https://github.com/wynddao/wynddex",
```

It points dex contract repo only. how about changing to point Wynd DAO organization?

```suggestion
    "github": "https://github.com/wynddao",
```

---

I think it depends on what I would like to emphasize; to highlight their community-driven efforts and their DAO system itself then WyndDAO would be better, but to put emphasis on the DeFi functionalities that are enabled by the application then WyndDEX would be better.

---

wdyt?

---

I don't think It's relevant to what we want to emphasize. It's more convenient to explore their code from `https://github.com/wynddao` because the code that drives the app is in multiple repos.